src/modules/course/dto/update-review.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import { ReviewType } from '../review.enum';
+
+export class UpdateReviewDto {
+  @IsNotEmpty()
+  @IsString()
+  reviewerName: string;
+
+  @IsNotEmpty()
+  @IsString()
+  reviewDescription: string;
+
+  @IsNotEmpty()
+  occupation: ReviewType;
```

Comment:
Try creating a request with occupation type other than student or teacher. It will accept it.
Add enum validator:
```
  @IsEnum(ReviewType)
```

Reply:
ok